bundles/org.openhab.binding.irobot/src/main/java/org/openhab/binding/irobot/internal/dto/MQTTProtocol.java

```diff
@@ -35,23 +34,30 @@
         public int ordered;
         @SerializedName("pmap_id")
         public String pmapId;
+        @SerializedName("user_pmapv_id")
+        public String userPmapvId;
         public List<Region> regions;
 
-        public CleanRoomsRequest(String cmd, String mapId, String[] regions) {
+        public CleanRoomsRequest(String cmd, String mapId, String[] pregions, String[] types, String user_mapv_id) {
```

Comment:
Naming convention: `userMapvId` instead of `user_mapv_id`.
As you are then setting `userPmapvId`, I guess the parameter should even be `userPmapvId`.